src/App.jsx

```diff
@@ -190,14 +154,23 @@ function App() {
 	if (errorState) {
 		return <ErrorMessage />;
 	}
-
+	function handleChildTabEvent(data) {
+		setSuggValidatorsData ({...data});
+		let individualStake = parseFloat(suggValidatorsData.budget)/16;
```

Comment:
@suyanksaraswat If this is never re-assigned, why not use `const`?

Reply:
'const' doesn't update the value. i.e I used 'let'.